### PR TITLE
Catch errors too

### PR DESCRIPTION
**Pull request by hmusum**

### CODE REVIEW

**Review by baldersheim — COMMENTED** [review]
👍

**Comment by bratseth** [comment]
I suggest you do         

} catch (Exception | NoClassDefFoundError e) {

instead, so we don't catch things where it would be better to die.

**Comment by hmusum** [comment]
Moved catching to make sure that we log something where thread that gets config will die anyway.

File: configserver/src/main/java/com/yahoo/vespa/config/server/rpc/GetConfigProcessor.java [hunk 1]
```diff
@@ -112,7 +112,7 @@ public void run() {
         } catch (UnknownConfigIdException e) {
             handleError(request, ErrorCode.ILLEGAL_CONFIGID, "Illegal config id " + request.getConfigKey().getConfigId());
             return;
-        } catch (Exception e) {
+        } catch (Exception | Error e) {
```
Comment (gjoranv) [inline configserver/src/main/java/com/yahoo/vespa/config/server/rpc/GetConfigProcessor.java]: I think you can just catch Throwable instead. In the (not very likely) event that a non-error throwable is thrown, the thread will die silently.

**Review by gjoranv — CHANGES_REQUESTED** [review]

File: configserver/src/main/java/com/yahoo/vespa/config/server/rpc/GetConfigProcessor.java (hunk 1 above)
Comment (hmusum) [reply configserver/src/main/java/com/yahoo/vespa/config/server/rpc/GetConfigProcessor.java]: fixed

**Review by gjoranv — APPROVED** [review]

**Pull request merged**